src/Uno.UWP/Security.Credentials/PasswordVault.Android.cs

```diff
@@ -60,41 +66,88 @@ public KeyStorePersister(string filePath = null)
 
 				if (store.ContainsAlias(_alias))
 				{
-					var key = store.GetKey(_alias, null);
-
-					_key = key;
+					if (Android.OS.Build.VERSION.SdkInt > BuildVersionCodes.LollipopMr1)
```

Comment:
As you are customizing all the methods of this class (which is actually an implementation of the Strategy Pattern)  ... you should instead create another strategy implementation (i.e. a new class which implements the `IKeyProvider`).

The only reusable code is the `KeyStore` resolution, which could easily be extracted in a static helper class.